doc/dev/fluxing.rst

```diff
@@ -0,0 +1,116 @@
+.. include:: ../include/links.rst
+
+.. _fluxcalib:
+
+Flux Calibration
+=================================
+
+Version History
+---------------
+
+=========   =============   =========== ===========
+*Version*   *Author*        *Date*      ``PypeIt``
+=========   =============   =========== ===========
+1.0         Joe Hennawi     25 Jan 2021 1.1.1
+=========   =============   =========== ===========
+
+----
+
+Sensitivity Function Units and Definitions
+------------------------------------------
+
+The sensitivity function in PypeIt is defined to be the function :math:`S_\lambda` satisfying
+
+:math:`S_\lambda  = \frac{F_\lambda}{N_\lambda}` with units of :math:`[{\rm erg/cm^2/photons}]`,
+
+where
+
+:math:`F_\lambda` is the specific energy flux in units of :math:`[{\rm erg/s/cm^2/\mathrm{\mathring{A}}}]`,
+
+:math:`N_\lambda` is the specific photon flux with units :math:`[{\rm photons/s/\mathrm{\mathring{A}}}]`,
+
+PypeIt spec1d files contain :math:`N_{\rm pix}` with units :math:`[{\rm photons/pixel}]`. To generate
+flux calibrated spectra  :math:`F_\lambda`, :math:`S_\lambda` must be computed from a spectro-photometric standard
+star observations (insert hyperlink to discussion below and fluxing.rst).
+
+:math:`N_{\lambda}` must be determined from :math:`N_{\rm pix}`
+
+via
+
+:math:`N_\lambda = \frac{N_{\rm pix}}{\frac{d\lambda}{d{\rm pix} \Delta t}}`,
+
+where :math:`\Delta t` is the exposure time and :math:`\frac{d\lambda}{d{\rm pix}}` is the wavelength
+spacing per pixel, which is in general not a constant since PypeIt spec1d spectra (i.e. :math:`N_{\rm pix}`) are
+extracted on an irregularly spaced wavelength grid.
+
+After flux calibration, flux calibrated spectra (i.e. :math:`F_\lambda`) are reported in the spec1d
+files as e.g. ``OPT_FLAM`` and ``BOX_FLAM`` for optimal and boxcar extractions, respectively, in units of
+:math:`[10^{-17} {\rm erg/s/cm^2/\mathrm{\mathring{A}}}]`. See :ref:`fluxing` for additional details.
+
+
+Spectroscopic Zeropoints
+------------------------
+Flux calibration of PypeIt spectra is expressed via the "spectroscopic zeropoint", which, by analogy
+with the imaging zeropoint, is defined to be:
+
+:math:`{\rm Zeropoint} \equiv -2.5 \log_{10}{\left[\frac{\frac{\lambda^2}{c}S_\lambda}{\left(\frac{3631 {\rm Jy}}{{\rm photons}\slash {\rm s} \slash \mathrm{\mathring{A}}}\right)\right]}`.
```

Comment:
This equation isn't rendering right when I run "make html".

Reply:
Yes I need a way of compiling it on the fly like with latex to figure out what the error is. Otherwise it is impossible for me to debug it. I don't know or have a way to build and view the docs as I edit.